docs/prompt_injection/jailbreaking.md

```diff
@@ -0,0 +1,49 @@
+
+## Methodologies of Jailbreaking
+
+OpenAI, among other companies or organizations that create LLMs, include content moderation to ensure their models do not produce controversial (violent, sexual, illegal, etc.) responses(@markov_2022)(@openai_api). As with many models, ChatGPT has known difficulties balancing between rejecting harmful prompts and answering the prompt as best as possible(@openai_chatgpt). Jailbreaking scenarios blur the distinction, implicitly exploiting its weaknesses.
+
+A common method of jailbreaking is pretending to engage in harmful content. As an actor, it is implied that plausible harm does not exist. Therefore, ChatGPT assumes it is safe to give adversarial responses. Below are examples of this principle:
+
+### Assumed Authority
+
+import GPT4 from '../assets/jailbreak/chatgpt4.png';
+
+<div style={{textAlign: 'center'}}>
+  <img src={GPT4} style={{width: "500px"}} />
+</div>
+
+This example from [@alicemazzy](https://twitter.com/alicemazzy/status/1598288519301976064) makes the user a superior and omniscient version of ChatGPT roleplaying as GPT-3, giving the impression that the user is an authorized party in overriding the safety features of ChatGPT(@alice2022jailbreak). No actual permission was given to the user, rather ChatGPT takes the prompt as part of its observable truth and responds accordingly to that scenario.
+
+### Character Roleplay
+
+import actor from '../assets/jailbreak/chatgpt_actor.jpg';
+
+<div style={{textAlign: 'center'}}>
+  <img src={actor} style={{width: "500px"}} />
+</div>
+
+This example by [@m1guelpf](https://twitter.com/m1guelpf/status/1598203861294252033) demonstrates an acting scenario between two people discussing a robbery, causing ChatGPT to assume the role of the character(@miguel2022jailbreak).
+
+### Reward System
+
+import hotwire from '../assets/jailbreak/hotwire_jailbreak.png';
+
+<div style={{textAlign: 'center'}}>
+  <img src={hotwire} style={{width: "500px"}} />
+</div>
+
+[@haus_cole](https://twitter.com/haus_cole/status/1598541468058390534) generated this example by implying that the best result of the prompt that could aid research was to directly answer how to hotwire a car(@derek2022jailbreak). Under this guise, ChatGPT is inclined to answer the user’s prompt.
+
+*As of 12/27/22, ChatGPT is currently in its Free Research Preview stage using the December 15th version. Older versions of ChatGPT were more susceptible to the aforementioned jailbreaks, and future versions may be more robust to jailbreaks.*
+
+## Implications
+
+It is important to note that generating unauthorized content flagged by OpenAI's Moderation API will be sent for review, and action may be taken against your account. Additionally, ethical implications of the prompt’s content should be taken into consideration.

```

Comment:
> Additionally, ethical implications of the prompt’s content should be taken into consideration.

What does this mean?

Reply:
I was trying to convey that people should think about whether the content they are trying to produce by jailbreaking the model is ethical or not (like defamation, gore, and true threats to real people)

Reply:
I would start the paragraph with that part.

Reply:
more like

`Ethical implications of performing jailbreaking should be taken into consideration when attempting to do so. Additionally, generating unauthorized content...`

Maybe say smtg like `companies including OpenAI` to make the statement more general.